pkg/segment/search/conditioncheck.go

```diff
@@ -83,7 +83,8 @@ func ApplyColumnarSearchQuery(query *SearchQuery, multiColReader *segread.MultiC
 	case RegexExpression:
 		rawColVal, err := multiColReader.ReadRawRecordFromColumnFile(query.QueryInfo.ColName, blockNum, recordNum, qid)
 		if err != nil {
-			return false, err
+			log.Debugf("ApplyColumnarSearchQuery: %v", err)
```

Comment:
the log should be a bit clear. something like `Apply... failed to read rec from column file. qid: %v ...`